src/makeAuth/index.tsx

```diff
@@ -35,7 +35,7 @@ function makeAuthenticator({
   placeholderComponent,
   signinArgs
 }: IMakeAuthenticatorParams) {
-  return <Props extends {}>(WrappedComponent: React.ReactNode) => {
+  return <Props extends {}>(WrappedComponent: React.ComponentType) => {
```

Comment:
I believe this should be:

```suggestion
  return <Props extends {}>(WrappedComponent: React.ComponentType<Props>) => {
```
This means that the component being wrapped expects the same props as those that will be passed to the component returned by this HoC.